api/src/services/extracts_v1/get_extracts.py

```diff
@@ -39,6 +40,22 @@ def get_extracts(
 ) -> Tuple[Sequence[ExtractMetadata], PaginationInfo]:
     stmt = select(ExtractMetadata)
 
+    # Apply sorting from pagination params
+    for sort_order in list_params.pagination.sort_order:
+        if sort_order.order_by == "extract_type":
+            # Join with lookup table for extract type description
+            stmt = stmt.join(
+                LkExtractType, ExtractMetadata.extract_type == LkExtractType.extract_type_id
+            )
+            sort_column = LkExtractType.description
+        else:
+            sort_column = getattr(ExtractMetadata, sort_order.order_by)
+
+        if sort_order.sort_direction == SortDirection.ASCENDING:
+            stmt = stmt.order_by(sort_column)
+        else:
+            stmt = stmt.order_by(desc(sort_column))
```

Comment:
Just for clarity - even if ascending is the default:
```suggestion
        if sort_order.sort_direction == SortDirection.ASCENDING:
            stmt = stmt.order_by(asc(sort_column))
        else:
            stmt = stmt.order_by(desc(sort_column))
```